app/models/game.rb

```diff
@@ -1,3 +1,12 @@
 class Game < ActiveRecord::Base
   enum status: [:safe, :check, :checkmate]
```

Comment:
inside game, we should has `has_many :pieces`

Then we can use `game.pieces` to get all of the pieces